src/flag_gems/ops/argmin.py

```diff
@@ -0,0 +1,176 @@
+import logging
+import math
+
+import torch
+import triton
+import triton.language as tl
+
+from ..utils import libentry
+from ..utils.shape_utils import can_use_int32_index
```

Comment:
Please follow the new changes in argmax and use `triton_lang_extension` since we made a decision to use int64 indexing everywhere to prevent unexpected overflow.

We used to use a more conservative way to do so by computing the maximum element offset of a tensor but now we decide to make it easier.

Reply:
DONE